docs/excel/excel-add-ins-ranges-get.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Get a range using the Excel JavaScript API
+description: 'Learn how to retrieve a range using the Excel JavaScript API.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Get a range using the Excel JavaScript API
+
+This article provides examples that show different ways to get a range within a worksheet using the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
+
+[!include[Excel cells and ranges note](../includes/note-excel-cells-and-ranges.md)]
+
+## Get range by address
+
+The following code sample gets the range with address **B2:C5** from the worksheet named **Sample**, loads its `address` property, and writes a message to the console.
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getItem("Sample");
+    var range = sheet.getRange("B2:C5");
+    range.load("address");
+
+    return context.sync()
+        .then(function () {
+            console.log(`The address of the range B2:C5 is "${range.address}"`);
+        });
+}).catch(errorHandlerFunction);
+```
+
+## Get range by name
+
+The following code sample gets the range named `MyRange` from the worksheet named **Sample**, loads its `address` property, and writes a message to the console.
```

Comment:
This feels like it should link to a section on naming ranges somewhere.

Reply:
Do we have a section on naming ranges? I can't find one.

Reply:
Seems like we only had the "get" part of named ranges. Let's create a low-priority work item to add the set named range conceptual section.

Reply:
Suggestion tracked with [internal work item](https://office.visualstudio.com/OC/_workitems/edit/5011021).